src/database/migrations/2023_02_13_221211_create_user_passports_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_passports', function (Blueprint $table) {
+            $table->id();
+            $table->string('passport_number', 32)->nullable()->comment('Номер паспорта');
+            $table->string('series', 32)->nullable()->comment('Серия паспорта');
+            $table->string('issued_by', 256)->nullable()->comment('Кем выдан');
+            $table->date('issued_date')->nullable()->comment('Когда выдан');
```

Comment:
Рано или поздно срок действия паспорта истекает и нам желательно знать не истек ли еще его срок действия
```suggestion
            $table->date('issued_date')->nullable()->comment('Когда выдан');
            $table->date('date_of_expiry')->comment('Дата истечения срока');
```
Зачем везде `nullable` это все необязательные данные?

Reply:
Добавил все поля что были описаны в тз)